workflows/theiacov/wf_theiacov_fasta.wdl

```diff
@@ -105,23 +331,28 @@ workflow theiacov_fasta {
     String? pangolin_docker = pangolin4.pangolin_docker
     String? pangolin_versions = pangolin4.pangolin_versions
     # Nextclade outputs
-    File? nextclade_json = nextclade.nextclade_json
-    File? auspice_json = nextclade.auspice_json
-    File? nextclade_tsv = nextclade.nextclade_tsv
-    String? nextclade_version = nextclade.nextclade_version
-    String? nextclade_docker = nextclade.nextclade_docker
-    String nextclade_ds_tag = nextclade_dataset_tag
+    File? nextclade_json = select_first([sarscov2_nextclade.nextclade_json, mpxv_nextclade.nextclade_json, rsv_a_nextclade.nextclade_json, rsv_b_nextclade.nextclade_json, flu_h1n1_ha_nextclade.nextclade_json, flu_h1n1_na_nextclade.nextclade_json, flu_h3n2_ha_nextclade.nextclade_json, flu_h3n2_na_nextclade.nextclade_json, flu_yam_ha_nextclade.nextclade_json, flu_vic_ha_nextclade.nextclade_json, flu_vic_na_nextclade.nextclade_json, create_dummy_file.outputFile])
```

Comment:
@jrotieno, instead of using a dummy_file task, just use "" instead and coerce this variable into a String. see also: https://github.com/theiagen/public_health_bioinformatics/blob/203ec3b580ed42bf0dd55566a639e8d0f7d63a1a/workflows/theiacov/wf_theiacov_illumina_pe.wdl#L315C3-L316C3

Reply:
Oh yes, this was before I discovered you could force a file into a String.

Reply:
I think one way we can reduce the redundancy in this task is to make a new task `set_organism_defaults` and then use those outputs for everything so we don't need to keep recalling the outputs, then we can go away with the select_firsts altogether!

Reply:
I might take a shot at this, if you don't mind

Reply:
Yes please :)

Reply:
I tried, it may have broken everything but I don't have time to test tonight 🙀